client/src/components/Navbar/Navbar.js

```diff
@@ -49,58 +51,74 @@ function Navbar() {
   return (
 		<>
 			<nav className="navbar">
-				<Link to="/" className="navbar-logo" onClick={closeMobileMenu}>
+				<NavLink to="/home" className="navbar-logo" onClick={closeMobileMenu}>
 					<img
 						src="https://i.postimg.cc/Z55sJJwz/Logo.jpg"
 						alt="sensational life logo"
 					></img>
-				</Link>
+				</NavLink>
 				<div className="menu-icon" onClick={handleClick}>
 					<i className={click ? "fas fa-times" : "fas fa-bars"} />
 				</div>
 				<ul className={click ? "nav-menu active" : "nav-menu"}>
```

Comment:
click is not good variable name, try to use more meaningful names for example
isClicked or whatever variable name represent